common/src/main/java/org/openmrs/analytics/FhirStoreUtil.java

```diff
@@ -90,9 +95,9 @@ public String executeRequest(HttpUriRequest request) {
 	
 	// This follows the examples at:
 	// https://github.com/GoogleCloudPlatform/java-docs-samples/healthcare/tree/master/healthcare/v1
-	public void uploadResourceToCloud(String resourceType, String resourceId, String jsonResource) {
```

Comment:
I commented above why we had the `String` based API before (instead of `Resource`). If you want to keep your new changes (which is fine) then I think it makes more sense to drop the `resourceType` and `resourceId` as both can be extracted from `resource`. It is fine if you want to leave this for a future PR but in that case please add a TODO; thanks.

Reply:
Yeah, I was struggling with the best approach, since I have this updated in my HAPI FHIR JPA work that I'll put in the next PR. I'll leave it as a TODO for now.